server/log_operation_manager.go

```diff
@@ -19,20 +19,39 @@ import (
 	"fmt"
 	"math/rand"
 	"reflect"
+	"strconv"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
 	"github.com/google/trillian/extension"
+	"github.com/google/trillian/monitoring"
 	"github.com/google/trillian/util"
 )
 
 const (
 	minPreElectionPause    = 10 * time.Millisecond
 	minMasterCheckInterval = 50 * time.Millisecond
 	minMasterHoldInterval  = 10 * time.Second
+	logIDLabel             = "logid"
 )
 
+var (
+	once         sync.Once
+	knownLogs    monitoring.Gauge
+	resignations monitoring.Counter
+	isMaster     monitoring.Gauge
+)
+
+func createMetrics(mf monitoring.MetricFactory) {
+	if mf == nil {
+		mf = monitoring.InertMetricFactory{}
+	}
+	knownLogs = mf.NewGauge("known_logs", "Set to 1 for known logs", logIDLabel)
```

Comment:
No, it's logs known about whether or not we're master (I've clarified the help).

My thought is that `known_logs` on the logsigner and the ctfe should be equal (for a CT-only Trillian instance).  
Likewise, if we sum `is_master` across instances it should always be 1 for each logID, and the sum across logIDs should equal the sum of `known_logs`.